get_http.go

```diff
@@ -220,6 +262,44 @@ func (g *HttpGetter) parseMeta(r io.Reader) (string, error) {
 	}
 }
 
+func GetByteRange(u *url.URL) ([]string, error) {
+	// format can be bytes=startByte- OR bytes=startByte-endByte
+	// example URL string: "http://my/file.iso?ranged_request_bytes=5555-66666"
+	// or  "http://my/file.iso?ranged_request_bytes=5555-"
+	// example bytes_range_retval: []int{5555, 66666} or []int{5555}
+	bytes_range_retval := make([]string, 0)
+	q := u.Query()
+	byteRange := q.Get("ranged_request_bytes")
+	if byteRange == "" {
+		return bytes_range_retval, fmt.Errorf("No byte range provided")
+	}
+	errMsg := fmt.Errorf("Invalid byte range provided")
```

Comment:
Since this is constant and reused multiple times, I recommend moving it to a constant var at the bottom of the file in package scope.

```
var errInvalidByteRange = fmt.Errorf("Invalid byte range provided")
```